pkg/apis/cluster/mutation/mutation.go

```diff
@@ -18,11 +18,20 @@ const (
 
 // MutateCluster mutates required fields of the Cluster.
 func MutateCluster(cluster *clusterapis.Cluster) {
-	MutateClusterTaints(cluster.Spec.Taints)
+	mutateClusterTaints(cluster.Spec.Taints)
+	mutateClusterZone(cluster)
 }
 
-// MutateClusterTaints add TimeAdded field for cluster NoExecute taints only if TimeAdded not set.
-func MutateClusterTaints(taints []corev1.Taint) {
+// mutateClusterZone add zones field for cluster if Zones not set but Zone set only.
+func mutateClusterZone(cluster *clusterapis.Cluster) {
```

Comment:
could you use a more appropriate function name? such as `setDefaultClusterZones`.

Reply:
How about the `MigrateZoneToZones()`, no big deal since it will be removed eventually.

Reply:
it's okay 😃